README.md

```diff
@@ -103,57 +103,138 @@ FLP is a framework. The main FLP object is the **pipeline**. FLP **pipeline** ca
+The second section provides the definition of specific configuration parameters for each one of the named stages.
+A full configuration file with the data consumed by  two different transforms might look like the following.
+
+```yaml
+pipeline:
+  - name: ingest1
+  - name: decode1
+    follows: ingest1
+  - name: generic1
+    follows: decode1
+  - name: write1
+    follows: generic1
+  - name: generic2
+    follows: decode1
+  - name: write2
+    follows: generic2
+parameters:
+  - name: ingest1
+    ingest:
+      type: file_loop
+      file:
+        filename: playground/goflow2_input.txt
+  - name: decode1
+    decode:
+      type: json
+  - name: generic1
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v1_bytes
+          - input: DstAddr
+            output: v1_dstAddr
+          - input: Packets
+            output: v1_packets
+          - input: SrcPort
+            output: v1_srcPort
+  - name: write1
+    write:
+      type: stdout
+  - name: generic2
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v2_bytes
+          - input: DstAddr
+            output: v2_dstAddr
+          - input: Packets
+            output: v2_packets
+          - input: SrcPort
+            output: v2_srcPort
+  - name: write2
+    write:
+      type: stdout
+```
 It is expected that the **ingest** module will receive flows every so often, and this ingestion event will then trigger the rest of the pipeline. So, it is the responsibility of the **ingest** module to provide the timing of when (and how often) the pipeline will run.
 
 # Configuration
 
 It is possible to configure flowlogs-pipeline using command-line-parameters, configuration file, environment variables, or any combination of those options.
 
+
 For example:
-1. Using command line parameters:
-```./flowlogs-pipeline --pipeline.ingest.type file --pipeline.ingest.file.filename hack/examples/ocp-ipfix-flowlogs.json```
-2. Using configuration file:
+1. Using configuration file:
 - create under $HOME/.flowlogs-pipeline.yaml the file:
 ```yaml
+log-level: info
 pipeline:
-  ingest:
-    type: file
-    file:
-      filename: hack/examples/ocp-ipfix-flowlogs.json
+  - name: ingest_file
+  - name: decode_json
+    follows: ingest_file
+parameters
+  - name ingest_file
+    ingest:
+      type: file
+      file:
+        filename: hack/examples/ocp-ipfix-flowlogs.json
+  - name: decode_json
+    decode:
+      type: json
 ```
 - execute `./flowlogs-pipeline`
+
+OR place the configuration information in any other file `<configFile>` and execute
+`./flowlogs-pipeline --config <configFile>`
+
+2. Using command line parameters:
+   ```./flowlogs-pipeline --log-level info --config <configFile>```
+Options included in the command line override the options specified in the config file.
```

Comment:
@KalmanMeth I am still not confident with that area of the README ... let's talk about this tomorrow. If the command line and environment variable options are broken we can add something temporal and add an issue to fix both ... but I do not want to have something that does not represent reality./

Reply:
The environment variables do not work currently. The command line does work as shown in the example.